### PR TITLE
Add `UI_GetDeployment` to `gateway.yml`

### PR DESCRIPTION
**Pull request by jgwhite**
## Why the change?

This has been missing for ages, suggesting we’re not currently using it. Still, doesn’t hurt to plug the gap.

### CODE REVIEW

File: pkg/server/gen/server.swagger.json [hunk 1]
```diff
@@ -6204,6 +6267,122 @@
         ]
       }
     },
+    "/ui/project/{project.project}/pipelines": {
```
Comment (jgwhite) [inline pkg/server/gen/server.swagger.json:6270]: Not strictly related to this change. Looks like we forgot to do a final `make gen/server` as part of #4509.

Comment (sdav9375) [reply pkg/server/gen/server.swagger.json:6270]: I'm a bit unclear on what happened. We did generate a bunch of stuff, so when was this step supposed to happen?

**Review by sdav9375 — APPROVED** [review]
Apart from being a bit unclear on the `pipelines` generated code, LGTM

**Comment by jgwhite** [comment]
@sdav9375 the takeaway (and this was new to me too) is that we need to run `make gen/server` whenever we change `gateway.yml`.

**Pull request merged**